content/docs/0.8.x/integrations/migrate_from_070_to_080/index.md

```diff
@@ -0,0 +1,739 @@
+| (Remediation) Action started                                            | `sh.keptn.event.action.started`                                        | `sh.keptn.event.action.started`                                                                            |
+| (Remediation) Action finished                                           | `sh.keptn.event.action.finished`                                       | `sh.keptn.event.action.finished`                                                                           |
+| Configure Monitoring                                                    | `sh.keptn.event.monitoring.configure`                                  | `sh.keptn.event.configure-monitoring` (pending changes)                                                    |
+| Configure Monitoring Finished                                           | -                                                                      | `sh.keptn.event.configure-monitoring.finished`                                                             |
+| Project was created                                                     | `sh.keptn.internal.event.project.create`                               | Ongoing discussion                                                                                                    |
+| Project was deleted                                                     | `sh.keptn.internal.event.project.delete`                               | Ongoing discussion                                                                                                    |
+| Service was created                                                     | `sh.keptn.internal.event.service.create`                               | Ongoing discussion                                                                                                    |
+| Service was deleted                                                     | `sh.keptn.internal.event.service.delete`                               | Ongoing discussion                                                                                                    |
+
+Please note, that even if the name stayed the same, the payloads might have changed. You will need to consult the respective CloudEvents doc, e.g.,
+* `action.triggered` on Keptn 0.7.x (spec 0.1.7): https://github.com/keptn/spec/blob/0.1.7/cloudevents.md#action-triggered
+* `action.triggered` on Keptn 0.8.x (spec 0.2.0): https://github.com/keptn/spec/blob/0.2.0/cloudevents.md#action-triggered
+
+
+## Workflow Changes (Shipyard)
+
+As already described above, we have significantly changed the way Keptn executes a sequence e.g. for continuous-delivery. Therefore we are providing some more guidance on how the workflow has changed in certain scenarios below.
```

Comment:
please change to: > As already described above, we have significantly changed the way Keptn executes a sequence, e.g., for continuous-delivery. Therefore, we are providing guidance on how the workflow has changed in certain scenarios below.